packages/calcite-components/src/components/stepper-item/stepper-item.tsx

```diff
@@ -210,11 +209,16 @@ export class StepperItem implements InteractiveComponent, LocalizedComponent, Lo
   render(): VNode {
     return (
       <Host
-        aria-expanded={toAriaBoolean(this.selected)}
+        aria-current={this.selected ? "step" : "false"}
         onClick={this.handleItemClick}
         onKeyDown={this.keyDownHandler}
       >
         <div class="container">
+          {this.complete && (
+            <span aria-live="polite" class="visually-hidden">
+              {"Completed step"}
```

Comment:
should reference message here.

Reply:
Without `T9nComponent` interface implemented, we cant test the message bundles for all supported locales. Once the translations are back this will be replaced.

Reply:
added #7759.

Reply:
This should reference the messages now right?